rest/src/main/java/com/netflix/conductor/rest/controllers/TaskResource.java

```diff
@@ -77,10 +77,10 @@ public ResponseEntity<List<Task>> batchPoll(
             @RequestParam(value = "workerid", required = false) String workerId,
             @RequestParam(value = "domain", required = false) String domain,
             @RequestParam(value = "count", defaultValue = "1") int count,
-            @RequestParam(value = "timeout", defaultValue = "100") int timeout) {
+            @RequestParam(value = "timeout", defaultValue = "0") int timeout) {
         // for backwards compatibility with 2.x client which expects a 204 when no Task is found
-        return Optional.ofNullable(
-                        taskService.batchPoll(taskType, workerId, domain, count, timeout))
+        // "long polling" is not working as expected
+        return Optional.ofNullable(taskService.batchPoll(taskType, workerId, domain, count, 0))
```

Comment:
`orkes-queues` requires changes and we need to fix this "long-polling" behaviour.